parm/config/gfs/config.resources

```diff
@@ -432,19 +435,27 @@ case ${step} in
       "C1152" | "C768")
         layout_x=8
         layout_y=8
+        walltime="00:45:00"
+        tasks_per_node=24
         ;;
       "C384")
         layout_x=6
         layout_y=6
+        walltime="00:30:00"
+        tasks_per_node=24
         ;;
       "C192" | "C96")
         layout_x=4
         layout_y=4
+        walltime="00:20:00"
+        tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
         ;;
       "C48" )
         # this case is for testing only
         layout_x=1
         layout_y=1
+        walltime="00:20:00"
+        tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
```

Comment:
Yes, this is much better.  However, `threads_per_task` needs to be defined before this case block so the `tasks_per_node` calculations at C48 and C96 aren't referencing undefined variables.

Reply:
whoops, duh, yes fixed now in latest commit